tendrl/commons/jobs/__init__.py

```diff
@@ -169,17 +164,14 @@ def process_job(job):
             )
             return
 
-        job_status_key = "/queue/%s/status" % job.job_id
-        job_lock_key = "/queue/%s/locked_by" % job.job_id
         try:
             lock_info = dict(node_id=NS.node_context.node_id,
                              fqdn=NS.node_context.fqdn,
                              tags=NS.node_context.tags,
                              type=NS.type)
-            etcd_utils.write(job_status_key, "processing",
```

Comment:
Please use these etcd_util writes here, we need that `prevValue="new"` check ,

make the `status` and `locked_by` watchable

Reply:
I can make status and locked_by watchable attributes I feel. Will try that.